DataRepo/models/msrun_sequence.py

```
@@ -18,6 +18,7 @@ class MSRunSequence(Model):
         ("QTOF", "QTOF"),
         ("unknown", "unknown"),
     ]
+    INSTRUMENT_DEFAULT = INSTRUMENT_CHOICES[4][0]
```

C. Note, this is not used as a "default" value for user data submissions.  It is used for 2 reasons (in 2 places):

1. To allow the validation page to proceed without complaining about a missing instrument value
2. As a placeholder value in order to proceed when a problem with an instrument value is encountered.  Whenever such a problem is encountered, an error is buffered and eventually raised at the end of a failed load.

So it's not really a "default" in the canonical sense.  If there's another word to use in the variable name that would be better, let me know.

Note, the `POLARITY_DEFAULT` in the file above is actually used as a default during a load.

---

- [ ] TODO: Add a comment to explain the use-cases for `INSTRUMENT_DEFAULT`